internal/config/config.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+import (
+	"path/filepath"
+	"strconv"
+
+	"github.com/btcsuite/btcutil"
+)
+
+var (
+	DefaultName    = "nigiri.config.json"
+	DefaultDatadir = btcutil.AppDataDir("nigiri", false)
+	DefaultPath    = filepath.Join(DefaultDatadir, DefaultName)
+
+	InitialState = map[string]string{
+		"attachliquid": strconv.FormatBool(false),
+		"datadir":      DefaultDatadir,
```

Comment:
can we remove datadir from the state? what's its usage for?

Reply:
just to keep up with old scherma. But yes not needed